### PR TITLE
[ENH, WIP] More tracking options

### PR DESCRIPTION
**Pull request by 36000**

### CODE REVIEW

File: AFQ/definitions/utils.py [hunk 1]
```diff
@@ -72,8 +72,8 @@ def _arglist_to_string(args, get_attr=None):
 def name_from_path(path):
     file_name = op.basename(path)  # get file name
     file_name = drop_extension(file_name)  # remove extension
-    if "-" in file_name:
-        file_name = file_name.split("-")[-1]  # get suffix if exists
+    if "_" in file_name:
```
Comment (arokem) [inline AFQ/definitions/utils.py:75]: Are we even using this function?

**Review by arokem — COMMENTED** [review]
What's the purpose of adding the bootstrap method here, if it's so slow as to be questionably useful?

**Comment by 36000** [comment]
We are going to try it for the baby data Howard is working with. We can remove it if it turns out not to be useful

**Comment by 36000** [comment]
It turns out ptt is also pretty slow, but thats only like two lines, so I think we can keep that.

File: AFQ/definitions/utils.py (hunk 1 above)
Comment (36000) [reply AFQ/definitions/utils.py:75]: ParticipantAFQ uses this function to generate names for scalars that the user inputs (such as a t1 image from another pipeline). I should make it so the user can also define a name, and we call this function only if a name is not provided.

**Pull request closed without merging**